src/tvheadend/Settings.h

```diff
@@ -38,6 +38,7 @@ namespace tvheadend {
     static const std::string DEFAULT_HOST;
     static const int         DEFAULT_HTTP_PORT;
     static const int         DEFAULT_HTSP_PORT;
+    static const bool         DEFAULT_USE_HTTPS;
```

Comment:
wrong intentation

Reply:
Slip of the eye. It compiles now, but I've not been able to test yet. Gotta get something x86 with X11 for that.